File: python/password_manager.py
class PasswordManager:
    def __init__(self):
        self.passwords = {}

    def add_password(self, site, password):
        self.passwords[site] = password

    def get_password(self, site):
        return self.passwords.get(site)

    def remove_password(self, site):
        if site in self.passwords:
            del self.passwords[site]

File: PasswordManager.java
import java.util.HashMap;

public class PasswordManager {
    private HashMap<String, String> passwords;

    public PasswordManager() {
        passwords = new HashMap<>();
    }

    public void addPassword(String site, String password) {
        passwords.put(site, password);
    }

    public String getPassword(String site) {
        return passwords.get(site);
    }

    public void removePassword(String site) {
        passwords.remove(site);
    }
}
